src/ImageSharp/Processing/Processors/Overlays/GlowProcessor.cs

```diff
@@ -65,25 +65,32 @@ protected override void OnApply(ImageBase<TPixel> source, Rectangle sourceRectan
                 startY = 0;
             }
 
+            int width = maxX - minX;
+            using (Buffer<TPixel> rowColors = new Buffer<TPixel>(width))
+            using (Buffer<float> amounts = new Buffer<float>(width))
             using (PixelAccessor<TPixel> sourcePixels = source.Lock())
             {
-                Parallel.For(
-                    minY,
-                    maxY,
-                    this.ParallelOptions,
-                    y =>
-                        {
-                            int offsetY = y - startY;
-                            for (int x = minX; x < maxX; x++)
-                            {
-                                int offsetX = x - startX;
-                                float distance = Vector2.Distance(centre, new Vector2(offsetX, offsetY));
-                                Vector4 sourceColor = sourcePixels[offsetX, offsetY].ToVector4();
-                                TPixel packed = default(TPixel);
-                                packed.PackFromVector4(Vector4BlendTransforms.PremultipliedLerp(sourceColor, glowColor.ToVector4(), 1 - (.95F * (distance / maxDistance))));
-                                sourcePixels[offsetX, offsetY] = packed;
-                            }
-                        });
+                for (int i = 0; i < width; i++)
+                {
+                    rowColors[i] = glowColor;
+                }
+
+                // TODO move GraphicOptions into core so all processes can use it.
+                PixelBlender<TPixel> blender = PixelOperations<TPixel>.Instance.GetPixelBlender(PixelBlenderMode.Normal);
+                for (int y = minY; y < maxY; y++)
```

Comment:
Why can this not be parallel? I see we are using the 3x faster parallel version in `Polaroid()` but this one for `Glow()`